docs/code/getting-started.md

```diff
@@ -422,6 +422,46 @@ Let's review what all we've learned to do:
 
 That's a lot! You're well on your way to becoming a bonified app developer for elementary OS. Give yourself a pat on the back, then take some time to play around with this example. Change the names of files and see if you can still build and install them properly. Ask another developer to branch your project from launchpad and see if it builds and installs cleanly on their computer. If so, you've just distributed your first app! When you're ready, we'll move onto the next section: Packaging.
 
+# Adding Translations {#Adding-Translations}
+Now that you've learned about CMake, the next step is to make your app able to be translated to different languages. The first thing you need to know is how to convert strings in your code into translatable strings.
+
+        stdout.printf ("Not Translatable string");
+        stdout.printf (_("Translatable string!"));
+        
+        string normal = "Another non-translatable string";
+        string translated = _(Another translatable string);
+
+See the difference? We just added `_()` around the string! Well, that was easy! Next you need to create a directory named "po" on the root folder of your project. Then you have to add the following lines in the "CMakeLists.txt" file you created a moment ago:
```

Comment:
I feel like there's a disconnect here between the example above and what we're actually asking the reader to do. We might want to make it clearer that we're asking the reading to make their existing project translatable. This is where we should break between the example and step 1 which would be, "go back to your project and make all your strings translatable"